app/workers/github/create_issue_job.rb

```diff
@@ -5,20 +5,27 @@ module Github
   class CreateIssueJob
     include Sidekiq::Worker
 
-    # limit the rate at which Github API calls are made
-    # so as not to trigger Github spam protections
-    sidekiq_options queue: 'low',
-                    rate: {
-                      name: 'prevent_feedback_spam',
-                      limit: 20,
-                      period: 3600, ## An hour
-                    }
+    # Rate limiter is only available in sidekiq-enterprise (which is used
+    # in production) & requires a license key. For development, regular sidekiq is used
+    THROTTLE =
+      if Rails.env.production?
+        Sidekiq::Limiter.window('prevent_feedback_spam', 4, :minute).freeze
```

Comment:
With the way I'm interpreting the docs, this is 4 per minute. That's different than the 20 per hour that this is replacing. Just want to double-check the rate here.

Reply:
20 per hour is a rate I made up initially.  I did more discovery since then and found:

- we once had 24/hour feedback submissions
- authenticated GH API calls limit us to 5k/hour
- un-authenticated calls limit to 60 per hour
- spamming GH API with identical `create_issue` calls, triggers their spam protection after ~65 attempts

I settled on 4/minute because:
- it won't slam GH with `create_issue` calls too quicky ([`sidekiq-ent` has a default backoff/retry strategy](https://github.com/mperham/sidekiq/wiki/Ent-Rate-Limiting#back-off))
- it leaves lots of room for growth since 24/hour was our highest
- ¯\_(ツ)_/¯

Reply:
It looks like some operations are subject to a lower (unspecified) limit, including "rapidly creating content": https://developer.github.com/v3/#abuse-rate-limits. 
which creating issues may count as. I'm okay with the current limits, agree monitoring and re-assessing is the way to go.